docs/smoothing.rst

```diff
@@ -72,6 +72,28 @@ the error persists, to then slowly increase epsilon until a valid common beam is
 found. More information can be found in the
 `radio-beam documentation <https://radio-beam.readthedocs.io/en/latest/>`_.
 
+Alternative approach to spatial smoothing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There is an alternative way to spatially smooth a data cube, which is using the
+:meth:`~spectral_cube.SpectralCube.spatial_smooth` method. Example code snippets
+are as follows::
```

Comment:
(1) there is only one code snippet below
(2) Instead of saying "example code snippet follows", say something about what the snippet does, i.e., "This is an example of how you do X":